mantle/cmd/kola/options.go

```diff
@@ -67,6 +67,7 @@ func init() {
 	sv(&kola.Options.CosaWorkdir, "workdir", "", "coreos-assembler working directory")
 	sv(&kola.Options.CosaBuildId, "build", "", "coreos-assembler build ID")
 	sv(&kola.Options.CosaBuildArch, "arch", coreosarch.CurrentRpmArch(), "The target architecture of the build")
+	sv(&kola.Options.InjectButane, "butane", "", "Path to Butane config which is merged with test code")
```

Comment:
For consistency, I wonder if we should support both `--butane` and `--ignition`. Obviously can be a follow-up.

Reply:
Bikeshed: should we name it something like `--butane-overlay` to better signal the fact that it doesn't replace the test config? Or maybe `--inject-butane` to match the internal name.

Reply:
I went with `--append-butane`.  In practice I think the docstring should be explanatory here.